src_blog/_posts/New-Release-Grain-v0.5-Durum.md

```diff
@@ -0,0 +1,97 @@
+---
+title: New Release! Grain v0.5 - Durum
+subtitle: Announcing the release of Grain v0.5, Durum, with faster compile times, big numbers, and more.
+date: 2022-06-06 12:00:00
+author: Oscar Spencer
+cover: /blog/photos/cover/yoav-aziz-pasta-unsplash.jpg
+coverAttribution: Yoav Aziz / Unsplash
+tags: Release, Changelog
+---
+
+Hey folks, today we’re absolutely thrilled to announce the release of Grain v0.5, Durum! This release marks an enormous step forward for the language and tooling, all thanks to the tremendous effort put in by the team and community to deliver a release that we're truly proud of. You can expect huge compiler performance improvements, better language ergonomics, and as always, better tooling. You can find the full [release notes](https://github.com/grain-lang/grain/releases/tag/grain-v0.5.0) on GitHub but we wanted to highlight some things we are excited about.
+
+## Language Features
+
+First up: the new language features included in this release.
+
+### Arbitrary precision math
+
+In Grain v0.5, you’ll find that number operations no longer overflow—you can create numbers as large as you like, only limited by the total memory of your Wasm host. Whether you need to do some data science or you just like really big numbers, Grain has your back.
```

Comment:
We technically don't have arbitrary precision floats; I'm worried someone will interpret this as we do. Thoughts?

Reply:
Eh, I'm not too worried about it. If someone comes asking for `BigFloat` we can address it then. I'll add "integer" to clarify.

Reply:
```suggestion
In Grain v0.5, you’ll find that number operations no longer overflow—you can create integer `Number`s as large as you like, only limited by the total memory of your Wasm host. Whether you need to do some data science or you just like really big numbers, Grain has your back.
```